seaworthy/pytest/fixtures.py

```diff
@@ -70,18 +71,48 @@ def test_container(postgresql):
         :mod:`seaworthy.definitions` module.
     :param name: The fixture name.
     :param scope: The scope of the fixture.
+    :param dependencies:
+        A sequence of names of other pytest fixtures that this fixture depends
+        on. These fixtures will be requested from pytest and so will be setup,
+        but nothing is done with the actual fixture values.
 
     :returns: The fixture function.
     """
     @pytest.fixture(name=name, scope=scope)
-    def fixture(docker_helper):
+    def fixture(request, docker_helper):
+        for dependency in dependencies:
+            request.getfixturevalue(dependency)
+
         definition.setup(helper=docker_helper)
         yield definition
         definition.teardown()
 
     return fixture
 
 
+def _pytest_fixture(self, name, scope='function', dependencies=()):
```

Comment:
Maybe call this `_definition_pytest_fixture`? Or, since we already know this is pytest stuff, `_definition_fixture`?